services/console-proxy/server/src/com/cloud/consoleproxy/ConsoleProxySecureServerFactoryImpl.java

```diff
@@ -112,8 +117,7 @@ public SSLServerSocket createSSLServerSocket(int port) throws IOException {
             SSLServerSocket srvSock = null;
             SSLServerSocketFactory ssf = sslContext.getServerSocketFactory();
             srvSock = (SSLServerSocket)ssf.createServerSocket(port);
-            srvSock.setEnabledProtocols(SSLUtils.getRecommendedProtocols());
-            srvSock.setEnabledCipherSuites(SSLUtils.getRecommendedCiphers());
```

Comment:
Let's keep the configured list of cipher suites. Check and fix this.

Reply:
done